app/webhook.go

```diff
@@ -107,7 +107,7 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 
 		a.Srv().Go(func() {
 			webhookResp, err := a.doOutgoingWebhookRequest(url, body, contentType)
-			if err != nil {
+			if err != nil && err != io.EOF {
```

Comment:
I'm slightly opposed to catching `io.EOF` at this layer, because it implies this caller has to know something about how the underlying object is consumed. Could we instead have `doOutgoingWebhookRequest` check for `io.EOF` and just return `nil, nil` in that case, clearly signalling there is no response to handle?

Reply:
Yep you are right, it is better to make the caller agnostic of the error type, I'll make the changes and I will add the unit tests ;)